app/gather/assets/apps/survey/mask/SurveyConfig.jsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from 'react'
+import { defineMessages, injectIntl } from 'react-intl'
+import { getLabelTree } from '../../utils/types'
+
+const createMessages = visualizations => {
```

Comment:
To do it properly (I mean in a i18n way) this could be done in this way (I don't mean it's the best option but one of them)

1. Remove `VISUALIZATIONS` array in `Survey` component and its propagation.

2. In `SurveyConfig`
```javascript

const MESSAGES = defineMessages({
  noChart: {
    id: 'survey.dashboard.config.no.chart',
    defaultMessage: 'No visualization',
  },
  barChart: {
    id: 'survey.dashboard.config.bar.chart',
    defaultMessage: 'Bar Chart'
  },
  pieChart: {
    id: 'survey.dashboard.config.pie.chart',
    defaultMessage: 'Pie Chart'
  },
  lineChart: {
    id: 'survey.dashboard.config.line.chart',
    defaultMessage: 'Line Chart'
  }
})

const VISUALIZATIONS = ['noChart', 'barChart', 'pieChart', 'lineChart']

const SurveyConfig = ({
  dashboardConfig,
  saveDashboardConfig,
  setShowConfig,
  columns,
  labels,
  intl: { formatMessage }
}) => {
  // ...
  // Replace any `messages` mention with `MESSAGES`
  // Replace any `visualizations` mention with `VISUALIZATIONS`
}

export default injectIntl(SurveyConfig)
```

3. Fix tests :wink: